content/blog/2020-07-10-july-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,150 @@
+
+- Plots powered by Vega-Lite so you can compare metrics across commits
+- New and easier pipeline configuration files- edit your DVC pipeline like a
+  text file!
+- Optimizations for data transfer speed
+
+Read all the [release notes](https://dvc.org/blog/dvc-1-0-release) for more, and
+stop by our [Discord](https://discordapp.com/invite/dvwXA2N) if you need support
+migrating (don't worry, 1.0 is backwards compatible).
+
+### DVC is in the top 20 fastest-growing open source startups
+
+Konstantin Vinogradov at [Runa Capital](https://runacap.com/) used the GitHub
+API to
+[identify the fastest growing public repositories on GitHub](https://medium.com/runacapital/open-source-growth-benchmarks-and-the-20-fastest-growing-oss-startups-d3556a669fe6)
+in terms of stars and forks. He used these metrics to estimate the top 20
+fastest growing startups in open source software. And guess what, DVC made the
+cut! We're in great company.
+
+![](/uploads/images/2020-07-10/top20startups.png)
+
+### New team member
+
+We have a new teammate-[Maxim Shmakov](https://www.linkedin.com/in/mvshmakov/),
+previously of Yandex, is joining us! Maxim is a front-end engineer joining us
+from Moscow. Please welcome him to DVC. 👋
+
+## Community activity
+
+We've been busy! Although we are mostly homebound these days, there has been no
+shortage of speaking engagements. Here's a recap.
+
+Co-founders Dmitry and Ivan appeared on the HasGeek TV series
+[Making Data Science Work](https://hasgeek.com/fifthelephant/making-data-science-work-session-3/)
+to discuss engineering for data science with hosts
+[Venkata Pingali](https://www.linkedin.com/in/pingali/) and
+[Indrayudh Ghoshal](https://www.linkedin.com/in/indrayudhghoshal/). The
+livestream is available for viewing on YouTube!
+
+https://www.youtube.com/watch?v=EWcpALbzZRg
+
+The same day, Dmitry appeared on the [MLOps.community](https://mlops.community/)
+meetup to chat with host
+[Demetrios Brinkmann](https://www.linkedin.com/in/dpbrinkm/). They talked about
+the open source ecosystem, the difference between tools and platforms, and what
+it means to codify data.
+
+https://www.youtube.com/watch?v=ojV1tK9jXH8&t=2295s
+
+I (Elle) gave another talk at the
+[MLOps Production & Engineering World](https://mlopsworld.com/) meeting, called
+"Adapting continuous integration and continuous delivery for ML". I shared an
+approach to using GitHub Actions & GitLab CI to start using continuous
+integration in ML model training- and it turns out, this conference is generally
+very excited about GitHub Actions! In another talk, GitHub technical advocate
+[Jonathan Peck](https://www.linkedin.com/in/peckjon/), shared some great use
+cases, including how he used an Nvidia Jetson Nano to run Actions. Generally,
+the mood around GitHub Actions in the ML community is very encouraging for a
+future with more automated data science practices.
+
+Extremely early the next morning, I co-led a workshop with clinician-scientist
```

Comment:
probably too much details to be honest :) and sounds a bit "dry" at this stage ... one talk after another after another

Reply:
yeah i can remove lots of detail